src/content/app/genome-browser/components/drawer/drawer-views/gene-summary/GeneSummary.tsx

```diff
@@ -112,6 +114,44 @@ const GeneSummary = () => {
         </div>
       </div>
 
+      <div className={classNames(rowClasses, styles.downloadRow)}>
+        <div className={styles.value}>
+          <ShowHide
+            label="Sequences"
+            isExpanded={shouldShowSequence}
+            onClick={() => showSequence(!shouldShowSequence)}
+          />
+        </div>
+        <div className={styles.value}>
+          {shouldShowSequence && (
+            <div className={styles.sequenceWrapper}>
+              <GeneSequenceView gene={gene} />
+            </div>
+          )}
+        </div>
+      </div>
```

Comment:
If we are going to merge this into dev before the sequence view is ready, we probably want to exclude this element when we build for production environment.

Reply:
Done